element_lab/export/nwb.py

```diff
@@ -1,7 +1,7 @@
 from element_lab import lab
 
 
-def lab_to_nwb_dict(lab_key: dict) -> dict:
+def _lab_to_nwb_dict(lab_key: dict) -> dict:
```

Comment:
Does your local deployment render underscored methods, or is that another setting we would need to toggle?

I made a comment thinking we could address this in the future, but I just looked through implementation elsewhere and I think this is a safe/compatible change to make

Reply:
I see that any function with the underscore prefix doesn't appear in the doc. I'll implement this for element-lab as well.